redbeams/src/main/java/com/sequenceiq/redbeams/service/crn/CrnService.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.redbeams.service.crn;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+
+@Service
+public class CrnService {
+
+    public Crn createDatabaseCrnFrom(String resource) {
+        return Crn.builder()
+                .setService(Crn.Service.REDBEAMS)
+                .setAccountId("ACCOUNT_ID")
```

Comment:
This seems like it needs to be updated later? Maybe add a "// FIXME" comment for it?

Reply:
Yes, done.